ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -248,12 +251,12 @@
   public static final String storeTtlUpdateBufferTimeSecondsName = "store.ttl.update.buffer.time.seconds";
 
   /**
-   * Specifies whether all index segments should be kept in memory (as opposed to a specific subset).
+   * Provides a hint for how indexes should be treated w.r.t memory
    */
-  @Config(storeKeepIndexInMemoryName)
+  @Config(storeIndexMemStateName)
   @Default("false")
```

Comment:
you need to change this.

Reply:
@vgkholla  Can you change the default value? Then I can merge this change.

Reply:
Fixed